core/tracepoint/enable_spec.rb

```diff
@@ -97,11 +97,54 @@
   end
 
   ruby_version_is "2.6" do
-    describe 'target: option' do
-      before :each do
-        ScratchPad.record []
+    before :each do
```

Comment:
The new specs should not be under the `ruby_version_is` guard (behavior is same before/after 2.6) but outside of it, with their own `ruby_bug` guard.